frontend/src/metabase/redux/app.ts

```diff
@@ -107,6 +112,25 @@ const isErrorDiagnosticsOpen = handleActions(
   false,
 );
 
+export const SET_TEMP_SETTING = "metabase/app/SET_TEMP_SETTING";
+const tempStorageReducer = (
```

Comment:
we use RTK nowadays for this. https://github.com/metabase/metabase/blob/6212151becf1d00c244e11e90a7ef63ab5a2d6e7/frontend/src/metabase/setup/reducers.ts#L27

Reply:
See that actions are created differently as well.
You might want to use `createSlice` instead of `createReducer`, it's up to you.

Reply:
Thanks.
`createSlice` was shorter.

https://github.com/metabase/metabase/pull/48987/commits/2b119fc501b352b70d97ed746b753b0a4f47528a